src/app/_components/Banner/Banner.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import { PAGE_MAX_WIDTH } from '@/common/constants/constants';
+import { Button } from '@/ui/Button';
+import { Text } from '@/ui/Text';
+import { Flex, FlexProps, Icon } from '@chakra-ui/react';
+import { X } from '@phosphor-icons/react';
+import { ReactNode, forwardRef, useEffect, useState } from 'react';
+
+export const Banner = forwardRef<
+  HTMLDivElement,
+  {
+    content: ReactNode;
+    isDismissible?: boolean;
+    bannerKey?: string;
+  } & Omit<FlexProps, 'content'>
+>(({ content, bannerKey, isDismissible, ...flexProps }, ref) => {
+  const [isDismissed, setIsDismissed] = useState(false);
+
+  useEffect(() => {
+    if (bannerKey) {
+      setIsDismissed(localStorage.getItem(bannerKey) === 'true');
+    }
+  }, [bannerKey]);
+
+  if (isDismissed) {
+    return null;
+  }
+
+  return (
+    <Flex
+      w="full"
+      minHeight={10}
+      alignItems="center"
+      justifyContent="center"
+      ref={ref}
+      py={1}
+      bg={{
+        base: 'var(--stacks-colors-accent-stacks-400)',
+        _dark: 'var(--stacks-colors-accent-stacks-600)',
+      }}
+      {...flexProps}
+    >
+      <Flex gap={3} maxWidth={PAGE_MAX_WIDTH} alignItems="center" justifyContent="center">
```

Comment:
Can you use breakpoint value for maxWidth? e.g. maxWidth="breakpoint-2xl", I'd love to eliminate this variable at some point.

Reply:
done